src/app/state/staking/selectors.ts

```diff
@@ -15,3 +16,17 @@ export const selectSelectedAddress = createSelector([selectStaking], state => st
 export const selectValidatorDetails = createSelector([selectStaking], state => state.selectedValidatorDetails)
 export const selectActiveDelegations = createSelector([selectStaking], state => state.delegations)
 export const selectDebondingDelegations = createSelector([selectStaking], state => state.debondingDelegations)
+
+const ValidatorStatusPriority = {
+  active: 1,
+  inactive: 2,
+}
+
+const sortValidatorsByStatus = (validators: Validator[]) => {
+  const array = [...validators]
+  return array.sort((a, b) => ValidatorStatusPriority[a.status] - ValidatorStatusPriority[b.status])
```

Comment:
I would have put this in parseValidatorsList, where we sort by escrow. And comment that it maintains sort by escrow because sort is stable.

Reply:
Scratch that. Let's move both sort(escrow).sort(status) to selectors.
(I don't think we will have API sorting validators and paginating them anytime soon, so this code doesn't need to be close to API)

Reply:
Logic moved to `parseValidatorsList` to keep `rank` logic for Monitor intact